qa/tasks/cephfs/test_volumes.py

```diff
@@ -6991,6 +7000,9 @@ def test_subvolume_snapshot_clone_cancel_pending(self):
         snapshot = self._gen_subvol_snap_name()
         clones = self._gen_subvol_snap_name(NR_CLONES)
 
+        # disable "capped" clones
```

Comment:
Is it required here, because its being disabled here: https://github.com/ceph/ceph/pull/55660/files#diff-d3a7e3f3f24fff510b4d2a562b2093257b20b3908748c3432d14e460c449186bR7016

Reply:
Sorry, I did not understand. The test required clones to be in pending state which the config would disallow by default.

Reply:
Ah, I see - will remove this hunk.